simpledotorg/simple-server
# Chartsjs upgrade to v3.9.1

**Story card:** [sc-9834](https://app.shortcut.com/simpledotorg/story/9834/update-chartjs-to-v3-9-1)

## Because

Updating ChartJS to the latest v3 (v3.9.1)
- Improved config (reduced bloat)
- Additional features
- Faster rendering

## This addresses

Unable to use newer features due to older version

## Test instructions

Graphs to check:
All HTN [bar and line]
All DM [bar and line]
Trending (small graph) [line]

Checking all graphs load 
Check hovering on all graphs works are expected - no errors thrown

I have done extensive checks on the functions that were updated to ensure the data point gathered is identical.
I have checked all graphs (and all data) on my localhost for a single region to ensure the data matches between this branch and the master
I have made checks to ensure the design is unchanged - this is the case in 98% of the design. There is some padding changes when upgrading.
